### PR TITLE
Package teasers

### PR DESCRIPTION
**Pull request by gvonkoss**

### CODE REVIEW

File: src/scss/themes/_hero.scss [hunk 1]
```diff
@@ -180,3 +180,54 @@
...
+
+	.o-teaser__heading {
+		color: #ffffff;
+	}
+
+	.o-teaser__standfirst {
+		background-color: oTeaserTextColor(#ffffff, $_o-teaser-color-extra-background, 80);
+	}
+
+	&.o-teaser--hero {
```
Comment (onishiweb) [inline src/scss/themes/_hero.scss]: Why is this required? Based on the way `oTeaserHeroExtra` is being included this will result in the selector `.o-teaser--extra-article.o-teaser--hero.o-teaser--hero`, is that the intended effect?

File: src/scss/themes/_hero.scss [hunk 2]
```diff
@@ -180,3 +180,54 @@
...
+			background-color: $_o-teaser-color-extra-background;
+			border-width: 0 1px 1px;
+			border-style: solid;
+			border-color: $_o-teaser-color-extra-highlights;
+		}
+
+		.o-teaser__content:after {
+			@include oGridRespondTo(S, L) {
+				background-color: $_o-teaser-color-extra-background;
+
```
Comment (onishiweb) [inline src/scss/themes/_hero.scss]: extra line

File: src/scss/themes/_hero.scss [hunk 3]
```diff
@@ -180,3 +180,54 @@
...
+		color: $_o-teaser-color-extra-highlights;
+	}
+	#{$o-hoverable-if-hover-enabled} .o-teaser__tag:hover,
+	.o-teaser__tag:focus {
+		color: oTeaserTextColor(#ffffff, $_o-teaser-color-extra-highlights, 73);
+	}
+
+	.o-teaser__meta {
+		color: $_o-teaser-color-extra-highlights;
+	 	&::after {
```
Comment (onishiweb) [inline src/scss/themes/_hero.scss]: single colon is ok here

File: src/scss/_mixins.scss [hunk 4]
```diff
@@ -183,15 +183,24 @@
...
 		@include oTeaserHeroThemeBase;
+	}
+
+	.o-teaser--opinion.o-teaser--hero {
 		@include oTeaserHeroOpinion;
 	}
 
-	.o-teaser--highlight.o-teaser--hero {
+	.o-teaser--highlight.o-teaser--hero,
+	.o-teaser--special-report-article.o-teaser--hero {
```
Comment (onishiweb) [inline src/scss/_mixins.scss]: Is it important to have this class if it's going to result in the same styles as `.o-teaser--highlight`? 

If it's not going to add anything, would it be possible to just use the highlight class in the presenter?

**Review by onishiweb — CHANGES_REQUESTED** [review]
This looks good. Just a couple of small points, let me know if you want to run through anything else.

File: src/scss/_mixins.scss [hunk 5]
```diff
@@ -183,15 +185,23 @@
 	}
 
 	.o-teaser--highlight.o-teaser--hero,
+	.o-teaser--special-report-article.o-teaser--hero,
```
Comment (onishiweb) [inline src/scss/_mixins.scss]: Sneaky special report class remains ;)

File: src/scss/_mixins.scss [hunk 6]
```diff
@@ -226,3 +236,12 @@
 		@include oTeaserDuration;
 	}
 }
+
+@mixin oTeaserThemePackage {
+	.o-teaser--basic-package,
+	.o-teaser--specia-report-package,
+	.o-teaser--extra-package {
+		@include oTeaserPackage;
+		@include packageTeaserList;
```
Comment (onishiweb) [inline src/scss/_mixins.scss]: Can this mixin be renamed to an `oTeaserX` mixin

File: src/scss/themes/_package.scss [hunk 7]
```diff
@@ -0,0 +1,87 @@
+@mixin oTeaserPackage {
+	@include oTeaserInverse;
+	width: 100%;
+
+	.o-teaser__image-placeholder {
```
Comment (onishiweb) [inline src/scss/themes/_package.scss]: What's the reason for the negative z-index?

File: src/scss/themes/_package.scss [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+
+	.o-teaser__meta {
+		padding: 15px 0 0 20px;
+		width: calc(100% - 40px);
+		position: relative;
+		top: -50px;
+		background: inherit;
+		color: $_o-teaser-color-basic-text;
+
+		&::after {
```
Comment (onishiweb) [inline src/scss/themes/_package.scss]: Only needs a single colon

File: src/scss/themes/_package.scss [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+		@include font-size(28, 30);
+		background: inherit;
+		padding: 20px;
+		margin-top: -50px;
+		width: 100%;
+		color: oTeaserTextColor(#000000, $_o-teaser-color-basic-text, 60);
+	}
+}
+
+@mixin packageTeaserList {
```
Comment (onishiweb) [inline src/scss/themes/_package.scss]: Mixin rename pls (mentioned above)

File: src/scss/themes/_package.scss [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+		&__list {
+			margin: -10px 0 0;
+			padding: 0 20px 10px;
+			background: inherit;
+			list-style: none;
+		}
+
+		&__list-item,
+		&__landing-link {
+			font-family: MetricWeb, sans-serif;
```
Comment (onishiweb) [inline src/scss/themes/_package.scss]: Can you use the `oTypography` mixins here for the family and font size/line-height. It will make migrating the styles across a lot easier in future

File: src/scss/themes/_package.scss [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+			padding: 10px 0;
+
+			> a {
+				text-decoration: none;
+				border: 0;
+			}
+		}
+
+		&__list-item a {
+			color: oTeaserTextColor(#000000, $_o-teaser-color-basic-text, 60);
```
Comment (onishiweb) [inline src/scss/themes/_package.scss]: Confused about these color settings, is your intention to mix black with the text colour? The result of this will be as if the text were black at 60% opacity on a background of the `$_o-teaser-color-basic-text`, is that correct?

**Review by onishiweb — CHANGES_REQUESTED** [review]
Couple more comments. 

Could you also add a demo for the new Package teaser and add the new theme and package teasers to the pa11y demo please

File: src/scss/themes/_package.scss (hunk 7 above)
Comment (gvonkoss) [reply src/scss/themes/_package.scss]: Oh—I forgot to mention. It's the only way I've manage to get the image behind the content for the package teasers ... :D

Comment (onishiweb) [reply src/scss/themes/_package.scss]: Ok cool, that makes sense 👍

**Review by onishiweb — APPROVED** [review]
